bodo/pandas/_plan.h

```diff
@@ -49,7 +60,10 @@ class BodoScanFunctionData : public duckdb::TableFunctionData {
  */
 class BodoParquetScanFunction : public BodoScanFunction {
    public:
-    BodoParquetScanFunction() : BodoScanFunction("bodo_read_parquet") {
+    BodoParquetScanFunction(const std::shared_ptr<arrow::Schema> arrow_schema)
+        : BodoScanFunction(
+              fmt::format("bodo_read_parquet({})",
+                          schemaColumnNamesToString(arrow_schema))) {
```

Comment:
Just improves plan prints for source nodes to include column names to allow noticing DuckDB's plan reorderings better.